### DISCUSSION TITLE
I did ti! Web framework based on Granian :)

### DISCUSSION THREAD

**Discussion by mcharytoniuk** [emmett-framework/granian#331] [Show and tell]
Just wrapped it up. It is a basic version (you can call it MVP :D), but it works. I wanted to innovate and used a lot of DI and role-based services approach (to make the most use possible out of async): https://github.com/distantmagic/intention

Let me know what you think. :D

Thank you for Granian!

**Comment by gi0baro** [comment]
I'm glad to see more frameworks taking advantage of RSGI :)
Congrats! 🚀
